spec/spec_helper.js

```diff
@@ -0,0 +1,24 @@
+import { JSDOM } from 'jsdom';
```

Comment:
I had hoped to try using [`happy-dom`](https://www.npmjs.com/package/happy-dom) as a faster alternative, but ran into a bug with custom elements (https://github.com/capricorn86/happy-dom/issues/958).